src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ui/HttpFuzzerResultsTableModel.java

```diff
@@ -262,7 +271,28 @@ public String getType() {
         public Map<String, Object> getCustomStates() {
             return customStates;
         }
-
+        
+        public List<Object> getFormat() {
+        	HttpFuzzerReflectionDetectorStateHighlighter value = new HttpFuzzerReflectionDetectorStateHighlighter();

```

Comment:
It can be just this line, maybe HttpFuzzerResultsTable? It's the class that actually uses FuzzResultStateHighlighter.